Library/Homebrew/cask/lib/hbc/utils.rb

```diff
@@ -145,8 +145,9 @@ def self.path_occupied?(path)
 
     def self.error_message_with_suggestions
       <<-EOS.undent
-        Most likely, this means you have an outdated version of Homebrew-Cask. Please run:
-          #{UPDATE_CMD}
+
```

Comment:
Without the newline the error message and the instructions get concatenated together. Would it be better to change line 155 in cli.rb to add a newline to the end of the error message?